apps/daimo-mobile/src/view/shared/Button.tsx

```diff
@@ -31,6 +33,7 @@ interface ButtonProps extends TextButtonProps {
 
 interface LongPressButtonProps extends ButtonProps {
   duration: number;
+  showBiometricIcon?: boolean;
```

Comment:
maybe unnecessary since we'll always show this on the long press button

could even rename to `SendButton`